src/Utils.ps1

```diff
@@ -176,6 +176,14 @@ function Add-PoshGitToProfile {
         $profileContent = "`nImport-Module '$ModuleBasePath\posh-git.psd1'"
     }
 
+    # Make sure the profile path exists
+    $profileDir = Split-Path -Parent $profilePath
+    if (-not (Test-Path -LiteralPath $profileDir)) { 
+        if ($PSCmdlet.ShouldProcess($profileDir, "Create current user Powershell profile directory")) {
+            New-Item $profileDir -Type Directory 
```

Comment:
Two things.  Better to use the actual parameter name `ItemType` than the alias `Type`.  Also, this command will output the created directory.  Better to just redirect the output to $null.  Then add support for -verbose e.g.:
```powershell
New-Item $profileDir -ItemType Directory -Verbose:$VerbosePreference > $null
```